turbo.json

```diff
@@ -5,10 +5,6 @@
 			"outputs": ["src/server/static/scripts/**", "src/server/static/styles/**"],
 			"dependsOn": ["^build", "$NODE_ENV", "$APP_RELEASE"]
 		},
-		"build:release": {
-			"outputs": ["src/server/static/scripts/**", "src/server/static/styles/**"],
-			"dependsOn": ["^build", "$NODE_ENV", "$APP_RELEASE"]
-		},
```

Comment:
This is actually still needed, this is the script used to build the web apps. The CI pipeline runs it, as do the Docker builds.

Reply:
cool - I've added it back